### PR TITLE
Builtin registries

### PR DESCRIPTION
**Pull request by YanisBft**
Maps the new dynamic registries
I know class_5458 is already covered by #1560 , it was just easier to name it before naming the other classes

### CODE REVIEW

**Review by Gegy — APPROVED** [review]

**Review by liach — APPROVED** [review]
Should be compatible with my pr after my rename

**Comment by liach** [comment]
After another look by earthcomputer that class actually stores the hardcoded values for dynamic registries

i.e. mojang now keep dynamic registries at work in registry trackers, while this class is used for registry tracker to build dynamic registries and fill in builtin values

File: mappings/net/minecraft/util/registry/DynamicRegistries.mapping [hunk 1]
```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/class_5458 net/minecraft/util/registry/DynamicRegistries
```
Comment (liach) [inline mappings/net/minecraft/util/registry/DynamicRegistries.mapping]: said to be `BuiltinRegistries` as there is a `missing built-in` message at where registry tracker uses  this class

**Review by liach — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/util/registry/DynamicRegistries.mapping (hunk 1 above)
Comment (YanisBft) [reply mappings/net/minecraft/util/registry/DynamicRegistries.mapping]: renamed

**Review by Juuxel — APPROVED** [review]

**Review by liach — APPROVED** [review]
Thank you for fixing my typos :heart:

**Pull request merged**